retriever/bin/retriever.js

```diff
@@ -121,7 +122,9 @@ export default {
           },
         }),
       )
-      return response
+      const clonedResponse = new Response(response.body, response)
```

Comment:
can we rename to `writableResponse`, to clarify the intend?

Reply:
sure, if we also update the comment in content-security-policy.js, which says it needs to be writable

Reply:
After trying different names, I decided to use a different approach:

- Rename to `fetch` response to `originResponse`
- Use just `response` for the Response objected returned by the worker